coreos-ostree-importer/README.md

```diff
@@ -99,3 +99,24 @@ You'll have to update the body with new information you'd like to use. Then run:
 ```
 ./publisher.py
 ```
+
+## Fedora Messaging sender from the Fedora CoreOS Pipeline
+
+Included in this directory is a file (`fcos-pipeline-ostree-import-request.py`)
+that is not used by the `coreos-ostree-importer`
+at all. It is used by the
+[Fedora CoreOS Pipeline](https://github.com/coreos/fedora-coreos-pipeline.git)
+to send the request to the importer. It made sense to co-locate the
+requester and importer in the same code repo/directory.
+
+Here's how you might send a request using `fcos-pipeline-ostree-import-request.py`: 
+
+
+```
+cosa buildprep --build=31.20200212.20.0 s3://fcos-builds/prod/streams/testing-devel/builds
+/usr/lib/coreos-assembler/fcos-pipeline-ostree-import-request.py \
+        --fedmsg-conf /srv/fedora-messaging-config.toml \
+        --build 31.20200212.20.0 --stg \
+        --s3 fcos-builds/prod/streams/testing-devel \
```

Comment:
This is actually written down by `buildprep` now: https://github.com/coreos/coreos-assembler/blob/5b8ff1fe481b125bc7965084eb345c95fd439ca1/src/cmd-buildprep#L53-L55. But yeah, I think still it's better to be explicit.